### DISCUSSION TITLE
Failing Unit Tests (WebApiTests.cs)

### DISCUSSION THREAD

**Discussion by davidnmbond** [simple-odata-client/Simple.OData.Client#845] [General]
@object, can you provide some advice on how best to get the failing unit tests (all in WebApiTests.cs) to pass?

There is nothing on the stated URL: http://va-odata-integration.azurewebsites.net/odata/open

I've tried https://www.odata.org/blog/advanced-odata-tutorial-with-simple-odata-client/ and must be missing something!

**Comment by object** [comment]
There was no way to get them pass because they were using a very old Web site that I set up almost 10 years ago and that is no longer there. There is a very little value in these tests, they were written before I started testing against Microsoft OData endpoints. So I deleted them all.

**Reply by davidnmbond** [reply]
Thanks!

**Comment by davidnmbond** [comment]
There are four unit tests with the [Skip] attribute applied:

- InsertPersonWithLinkToPeople
- InsertPersonWithLinkToMe
- BatchPayloadRelativeUri
- BatchPayloadAbsoluteUriUsingHostHeader

Any opinions on the fixes or desire to get them passing?  If not, I'll take a look, but it may be a few days with the company wind-down to Christmas...
